components/memix/actions/random/random.mjs

```diff
@@ -0,0 +1,29 @@
+import { axios } from "@pipedream/platform";
```

Comment:
instead of importing axios here, you should import the app file e.g:  
`import app from "../../memix.app.mjs";`

Reply:
i see. i've refactored to do that